### PR TITLE
feat: create reusable setup instructions

### PR DESCRIPTION
**Pull request by cjdcordeiro**
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

Replace the "Prerequisites" section, in all tutorials, with a reusable rst file.

### CODE REVIEW

File: docs/reuse/tutorials/setup.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+In order to create the rock, you'll need to install Rockcraft:
+
+.. literalinclude:: /reuse/tutorials/code/task.yaml
+    :language: bash
+    :start-after: [docs:install-rockcraft]
+    :end-before: [docs:install-rockcraft-end]
+    :dedent: 2
+
+We'll use Docker run the rock. You can install it as a ``snap``:
```
Comment (linostar) [inline docs/reuse/tutorials/setup.rst:51]:
```suggestion
We'll use Docker to run the rock. You can install it as a ``snap``:
```

**Review by linostar — CHANGES_REQUESTED** [review]

File: docs/reuse/tutorials/code/task.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+execute: |
+  date
+  # [docs:install-rockcraft]
+  sudo snap install rockcraft --classic
+  # [docs:install-rockcraft-end]
+
+  # [docs:install-docker]
+  sudo snap install docker
+  # [docs:install-docker-end]
```
Comment (cjdcordeiro) [inline docs/reuse/tutorials/code/task.yaml:18]:
```suggestion
  # [docs:install-docker-end]

```

File: docs/reuse/tutorials/setup.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    Is Multipass_ already installed and active? Start by running
+
+    .. code-block:: bash
+
+        sudo snap services multipass
+
+    If you see the ``multipass`` service but it isn't "active", then you'll
+    need to run ``sudo snap start multipass``. On the other hand, if you get
+    an error saying ``snap "multipass" not found``, then you must Install
+    `Multipass <install-multipass_>`_:
```
[the remark this answers is not in the document]
Comment (cjdcordeiro) [reply docs/reuse/tutorials/setup.rst:15]: hum I'd rather keep it consistent with the other multipass links in this page

[the remark this answers is not in the document]
Comment (linostar) [reply docs/reuse/tutorials/setup.rst:15]: In that case you probably want to lowercase `Install`.

[the remark this answers is not in the document]
Comment (cjdcordeiro) [reply docs/reuse/tutorials/setup.rst:15]: true

File: docs/reuse/tutorials/setup.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    Is Multipass_ already installed and active? Start by running
+
+    .. code-block:: bash
+
+        sudo snap services multipass
+
+    If you see the ``multipass`` service but it isn't "active", then you'll
+    need to run ``sudo snap start multipass``. On the other hand, if you get
+    an error saying ``snap "multipass" not found``, then you must Install
```
Comment (cjdcordeiro) [inline docs/reuse/tutorials/setup.rst:14]:
```suggestion
    an error saying ``snap "multipass" not found``, then you must install
```

File: docs/reuse/tutorials/setup.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (cjdcordeiro) [reply docs/reuse/tutorials/setup.rst]: Done

**Review by linostar — APPROVED** [review]

**Review by tigarmo — APPROVED** [review]

**Review by sergiusens — APPROVED** [review]

**Pull request merged**